app/controllers/application_controller.rb

```diff
@@ -102,7 +102,8 @@ def authorization_error msg, ref
     error += "<br />If you believe this is in error please contact, #{I18n.t 'error_contact'}, and provide the following information:"
     error += "<br /> Reference #: "
     error += ref
-    render 'service_requests/authorization_error', error: error
+    
+    render partial: 'service_requests/authorization_error', locals: { error: error, in_dashboard: false }
```

Comment:
For some reason I haven't been able to get the new syntax (without partial: and locals:) working in controllers. For the meantime, use old syntax